client/lib/analytics/ad-tracking.js

```diff
@@ -182,6 +186,39 @@ function recordPurchase( product ) {
 	} );
 }
 
+/**
+ * For Atlas, we track the entire order, not separately for each product in the order
+ *
+ * @see https://app.atlassolutions.com/help/atlashelp/727514814019823/ (Atlas account required)
+ *
+ * @param {Object} cart - cart as `CartValue` object
+ */
+function recordOrderInAtlas( cart ) {
+	const currentUser = user.get();
+	let params, urlParams, urlWithParams;
```

Comment:
Not necessary to change, but something that might interest you: identifiers declared with `let` are not hoisted like those with `var` are, so you could declare these as needed throughout the function block (and use `const` because there's no reassignment if you do this).